src/commands/config.ts

```diff
@@ -0,0 +1,124 @@
+import { SlashCommandBuilder } from '@discordjs/builders';
+import {
+    APIApplicationCommandOptionChoice,
+    PermissionFlagsBits,
+} from 'discord-api-types/v9';
+import { CommandInteraction, GuildMember } from 'discord.js';
+import { Config, ConfigKey } from '../utils/config';
+import { Embed } from '../utils/embed';
```

Comment:
Could you please export classes created in `utils` using `index.ts`? This will lead to much cleaner code
```suggestion
import { Config, ConfigKey, Embed } from '../utils';
```